### ISSUE TITLE
Some cell rows don't have a bottom border

### ISSUE THREAD

**Issue by vu0tran** [Purii/react-native-tableview-simple#32]
Take a look under the Obs Unit Protocols cell. It seems to be only happening on certain android devices. For example, this is the Nexus 5x. I can't seem to repro it on iOS or on Xiaomi mi 5, or Samsung Galaxy 3. 

![image](https://cloud.githubusercontent.com/assets/1264648/25219590/3d1d004c-25af-11e7-83f7-9a17cc7b099a.png)

**Comment by Purii** [comment]
I experienced this issue before, but only on iOS devices.
There you won't get this on a real device. I solved it, by setting the scale to 100% on the iOS Simulator. (Simulator: Window > Scale > 100%)

Could this be a similar issue on android simulator?

**Comment by vu0tran** [comment]
@Purii That's what I thought initially too. On iOS it's because those are rendered at 0.5 pixels on retina, and sometimes on simulator they'll round to 0.

But I don't think it's the case for Android Here's it really zoomed in and it seems to be happening on device too.

![screenshot_1493045691](https://cloud.githubusercontent.com/assets/1264648/25343239/c22c3642-290e-11e7-8c8c-3d8c7f9e06cc.png)

**Comment by Purii** [comment]
Spontaneously I do not have an idea, how to solve that issue. Maybe a problem with `StyleSheet.hairlineWidth`..
Could you upload an example, please?

**Comment by neiker** [comment]
same problem on ios

**Comment by hawkrives** [comment]
I think I have a minimal reproduction. I'll try to upload it this weekend.

**Comment by drewvolz** [comment]
[On line 93 of Section.js](https://github.com/Purii/react-native-tableview-simple/blob/master/components/Section.js#L93) there is this a check to see if the index is equal to the number of children.

```js
// Skip rendering of separator
if (
  hideSeparator ||
  !Array.isArray(children) ||
  children.length === 1 ||
  index === children.length - 1
) {
  return React.cloneElement(child, propsToAdd);
}
```

Here is the offending line in particular

```js
index === children.length - 1
```

If we take out that check, I do not see any cells that are missing a bottom border.

**Comment by Purii** [comment]
Could someone upload an example, please?

**Comment by hawkrives** [comment]
@Purii I finally made one!

```js
import React from 'react'
import {ScrollView, View} from 'react-native'
import {TableView, Section, Cell} from 'react-native-tableview-simple'

const Example1 = () =>
  <TableView>
    <Section header="BASIC REPRO">
      {[0, 1].map(i => <Cell key={i} title={i} />)}
      <Cell title="Add More Hours" />
      <Cell title="Delete" />
    </Section>
  </TableView>

const Example2 = () =>
  <TableView>
    <Section header="NO MAP">
      <Cell title={0} />
      <Cell title={1} />
      <Cell title="Add More Hours" />
      <Cell title="Delete" />
    </Section>
  </TableView>

export default () =>
  <ScrollView>
    <Example1 />
    <Example2 />
  </ScrollView>
```

produces

<img width="320" alt="screen shot 2017-08-17 at 3 24 58 pm" src="https://user-images.githubusercontent.com/464441/29432189-47407e82-8360-11e7-92fa-1772f7099f17.png">

---

I would have expected both of those to have the line above "delete", and it also looks like the first example has an extra line under the delete button.

If you need something else, please let me know!

**Comment by hawkrives** [comment]
Thanks @Purii!

**Comment by Purii** [comment]
Should be fixed with v0.16.9 👍
